### ISSUE TITLE
Drop the default navigation bar accent color

### ISSUE THREAD

**Issue by meel-hd** [WeblateOrg/weblate#13321]
### Describe the problem

The current dark blue default navigation color makes the app look old and outdated.
![Screenshotsadf17 145332](https://github.com/user-attachments/assets/a36df838-5595-4927-b085-4c0b9228d8f8)

### Describe the solution you would like

What if we removed that color and made the default the same as the UI background color. This way the UI looks fresh and modern.
I know there is the option to customize colors in `/manage/appearance` but I'm talking that we have this as the default when there's no custom colors.

### Screenshots

![Screenshot 2024-12-16 184503](https://github.com/user-attachments/assets/ae375a96-96ab-479c-9f8f-b0179155e353)
![Untitled](https://github.com/user-attachments/assets/6a027609-b77a-4073-a691-394bc01294f3)

### Additional context

Go to `/manage/appearance` and change:
- the navigation background to be white  and the text black for light mode.
- and for dark mode the background of the navigation to be the same as the background of the UI. and the text also.

@nijel @Geeyun-JY3 @BoFFire @yarons what do you think?

**Comment by nijel** [comment]
I don't see anything outdated on this.

**Comment by yarons** [comment]
It's a nice suggestion overall but I think a bigger overhaul required.

**Comment by nijel** [comment]
I would like to do a bigger UI overhaul next year, but it all depends on funding, which is still not yet clear.

**Comment by nijel** [comment]
@orangesunny What do you think about this change?

**Comment by Angeschossen** [comment]
I don't think this change would serve usability well. You would need at least a border to separate the header from the main content.

**Comment by meel-hd** [comment]
@Angeschossen I guess yes. No color at all would be confusing, btw I drew inspiration from the change Github made last year. They had similar dark navigation bar and they changed it to the greyish one available now. It made it easy on they eye and seem less cluttered (atleast in my view).

**Comment by nijel** [comment]
Let's stick with the current UI for now.
